style.css

```diff
@@ -0,0 +1,128 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+:root {
+    /***** gradients *****/
+    --scissors-gradient-one: hsl(39, 89%, 49%);
+    --scissors-gradient-two: hsl(40, 84%, 53%);
+    --paper-gradient-one: hsl(230, 89%, 62%);
+    --paper-gradient-two: hsl(230, 89%, 65%);
+    --rock-gradient-one: hsl(349, 71%, 52%);
+    --rock-gradient-two: hsl(349, 70%, 56%);
+    --lizard-gradient-one: hsl(261, 73%, 60%);
+    --lizard-gradient-two: hsl(261, 72%, 63%);
+    --cyan-gradient-one: hsl(189, 59%, 53%);
+    --cyan-gradient-two: hsl(189, 58%, 57%);
+    /***** text *****/
+    --dark-text: hsl(229, 25%, 31%);
+    --score-text: hsl(229, 64%, 46%);
+    --header-outline: hsl(217, 16%, 45%);
+    font-family: 'Barlow Semi Condensed', sans-serif;
+    /***** background *****/
+    --background-gradient-one: hsl(214, 47%, 23%);
+    --background-gradient-two: hsl(237, 49%, 15%);
+}
+html {
+    height: 100%;
+    background-size: cover;
+    /* background-size: 100%; */
+    background-repeat: no-repeat;
+    background-image:radial-gradient(ellipse at center top, var(--background-gradient-one), var(--background-gradient-two));
+
+}
+body {
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 20px;
+}
+button {
+    border: none;
+}
+
+/***** header *****/
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 20px;
+    width: 600px;
```

Comment:
The width is not responsive, right now. can you change that width to something more responsive? You can work with percentage for example (or whatever you find comfortable to work with)

Reply:
I don't think percentages will be good here. 
I would consider using `rem` though for different reasons, such as, it would be much easier to scale the site using rem, and also it would help make the site more accessible since users that are using the web with some zoom will have the right ratio of things if it will be based on the root element font size

Reply:
Ok I will change the units to rem